polylogue/document_store.py

```
@@ -0,0 +1,416 @@
+
+        try:  # pragma: no cover - indexing failures shouldn't abort writes
+            update_index(
+                provider=provider,
+                conversation_id=conversation_id,
+                slug=slug,
+                path=markdown_path,
+                document=document,
+                metadata=polylogue_meta,
+            )
+        except Exception:
+            pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)